spec/features/flowers/index_spec.rb

```diff
@@ -40,10 +40,11 @@
   it 'shows all flowers names' do
     visit "/flowers"
 
-
     expect(page).to have_content(@flower1.species)
-    # expect(page).to have_content("Shelf Life: 3 Days")
-    # expect(page).to have_content("In Stock: false")
```

Comment:
Did these tests get moved somewhere else or are you waiting to test them later? I think we should probably have tests for these somewhere

Reply:
Adding in the next PR.